src/app/loginComponents/accounts/external/accounts.component.ts

```diff
@@ -188,6 +193,9 @@ export class AccountsExternalComponent implements OnInit, OnDestroy {
     this.tokenQuery.tokens$.subscribe((tokens: Token[]) => {
       this.setTokens(tokens);
     });
+    this.userQuery.user$.subscribe((user: User) => {
```

Comment:
don't subscribe, pipe map. this way, if user.orcid isn't needed, it won't be subscribed to